src/plays/codenchill/styles.css

```diff
@@ -0,0 +1,67 @@
+/* Background color styling */
+.color-change-5x {
+  -webkit-animation: color-change-5x 10s linear infinite alternate both;
+  animation: color-change-5x 10s linear infinite alternate both;
+}
+
+@keyframes color-change-5x {
+  0% {
+    background: #90f4bd;
+  }
+  25% {
+    background: #fff494;
+  }
+  50% {
+    background: #ffbfe3;
+  }
+  75% {
+    background: #a6ddf3;
+  }
+  100% {
+    background: #f4cd9e;
+  }
+}
+
+/* Scrollbar styling */
+::-webkit-scrollbar {

```

Comment:
We need to avoid generic pseudo selectors to style specific play. It would impact the platform. Can you avoid this one?